pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2137,3 +2216,46 @@ func addAppInstanceOverlayRoute(
 			status.DisplayName)
 	}
 }
+
+// getSSHPublicKeys : returns decrypted SSH public keys
```

Comment:
Decrypted is clearly wrong. Trusted SSH public keys.

If we are doing to keep this function (which I question) we should at least add a comment saying that we don't expect EVE to ever have any trusted public keys to hand to an app instance since EVE does not run openstack.

Reply:
Corrected. 
I think, we can do introduce ssh public keys uploading through controller in the future the same way as it done for abstract userData.

Reply:
But everybody in the Linux world already does that as part of user-data AFAIK.